Service/Dumper.php

```diff
@@ -234,10 +236,20 @@ protected function deleteExistingSitemaps($targetDir)
     }
 
     /**
-     * @inheritdoc
+     * Create new DumpingUrlset with gz option
+     *
+     * @param string file name 
+     * @param DateTime|null date of last modify
+     * @param bool gx extention option
+     *
+     * @return DumpingUrlset
      */
-    protected function newUrlset($name, \DateTimeInterface $lastmod = null)
+    protected function newUrlset($name, \DateTimeInterface $lastmod = null, bool $gzExtention = false): DumpingUrlset
```

Comment:
Typo in the word "extention". To be replaced with "extension".

Reply:
Please, do not add strict return types here (will do in a later and dedicated PR : globally)